koku/masu/test/util/azure/test_azure_post_processor.py

```diff
@@ -44,9 +44,9 @@ def test_azure_process_dataframe(self):
             result, _ = self.post_processor.process_dataframe(df)
             columns = list(result)
             expected_columns = sorted(
-                col.replace("-", "_").replace("/", "_").replace(":", "_").lower() for col in TRINO_COLUMNS
+                col.replace("-", "_").replace("/", "_").replace(":", "_").lower() for col in TRINO_REQUIRED_COLUMNS
```

Comment:
You could use a translation table here instead of chained `replace` calls.
```suggestion
            translation = str.maketrans({char: "_" for char in "-/:"})
            expected_columns = sorted(
                col.translate(translation).lower() for col in TRINO_REQUIRED_COLUMNS
```